docs/basics/101-130-yodaproject.rst

```diff
@@ -0,0 +1,600 @@
+- The subdataset is linked to the superdataset thanks to
+  :command:`datalad install -d`.
+- The :command:`datalad run` command took care of linking the outputs of your
+  analysis with the script and the input data it was generated from.
+
+Let's take a look at the history of the ``midterm_project`` analysis
+dataset:
+
+.. runrecord:: _examples/DL-101-130-112
+   :language: console
+   :workdir: dl-101/DataLad-101/midterm_project
+
+   $ git log
+
+"Wow, this is so clean an intuitive!" you congratulate yourself. "And I think
+this was and will be the fastest I have ever completed a midterm project!"
+But what is still missing is a human readable description of your dataset.
+The YODA procedure kindly placed a ``README.md`` file into the root of your
+dataset that you can use for this [#f4]_.
+
+.. runrecord:: _examples/DL-101-130-113
+   :language: console
+   :workdir: dl-101/DataLad-101/midterm_project
+
+   # with the >| redirection we are replacing existing contents in the file
+   $ cat << EOT >| README.md
+
+   # Midterm YODA Data Analysis Project
+
+   ## Dataset structure
+
+   - All inputs (i.e. building blocks from other sources) are located in input/.
+   - All custom code is located in code/.
+   - All results (i.e., generated files) are located in the root of the dataset:
+     - "prediction_report.csv" contains the main classification metrics.
+     - "output/pairwise_relationships.png" is a plot of the relations between features.
+
+   EOT
+
+.. runrecord:: _examples/DL-101-130-114
+   :language: console
+   :workdir: dl-101/DataLad-101/midterm_project
+
+   $ datalad status
+
+.. runrecord:: _examples/DL-101-130-115
+   :language: console
+   :workdir: dl-101/DataLad-101/midterm_project
+
+   $ datalad save -m "Provide project description" README.md
+
+
+Note that one feature of the YODA procedure was that it configured certain files
+(for examples everything inside of ``code/`` and the ``README.md`` file in the
+root of the dataset) to be saved in Git instead of Git-annex. This was the
+reason why the ``README.md`` in the root of the dataset was easily modifiable [#f4]_.
+
+.. findoutmore:: Saving contents with Git regardless of configuration with --to-git
+
+   .. index:: datalad command; save --to-git
+
+   The ``yoda`` procedure in ``midterm_project`` applied a different configuration
+   within ``.gitattributes`` than the ``text2git`` procedure did in ``DataLad-101``.
+   Within ``DataLad-101``, any text file is automatically stored in :term:`Git`.
+   This is not true in ``midterm_project``: Only the existing ``README.md`` files and
+   anything within ``code/`` are stored -- everything else will be annexed.
+   That means that if you create any other file, even text files, inside of
+   ``midterm_project`` (but not in ``code/``), it will be managed by :term:`Git-annex`
+   and content-locked after a :command:`datalad save` -- an inconvenience if it
+   would be a file that is small enough to be handled by Git.
```

Comment:
Maybe mention that such configuration scenarios can be combined: `create -c text2git -c yoda`

Reply:
I didn't even know this, thx

Reply:
Actually, I will add this to the chapter on configurations.